lua/fishing_mod/cl_shop_menu.lua

```diff
@@ -235,6 +421,148 @@ end
 vgui.Register("Fishingmod:BaitShop", PANEL, "DPanelList")
 
 
+-- Customization tab
+local PANEL = {}
+
+function PANEL:Init()
+	local x, y = self:GetParent():GetSize()
+	self:SetSize(x - 6, y - 3 - 46)
+	self:SetPos(3, 46)
+	self:SetVisible(false)
+	function self:Think()
+		x, y = self:GetParent():GetSize()
+		self:SetSize(x - 6, y - 3 - 46)
+		if fishingmod.ColorTable then
+			back_ground = fishingmod.ColorTable.ui_background or back_ground
+			ui_text = fishingmod.ColorTable.ui_text or ui_text
+			button_not_selected = fishingmod.ColorTable.ui_text_bg or button_not_selected
+			ui_button_hovered = fishingmod.ColorTable.ui_button_hovered or ui_button_hovered
+			nopres = fishingmod.ColorTable.ui_button_deselected or nopres
+			pres = fishingmod.ColorTable.ui_button_pressed or pres
+		else
+			fishingmod.ColorTable = fishingmod.DefaultUIColors()
+		end
+	end
+
+	local savebutton = vgui.Create("DButton", self)
+	savebutton:SetPos(10, 50)
+	savebutton:SetSize(120, 30)
+	savebutton:SetTextColor(ui_text)
+	savebutton:SetText("Save")
+	savebutton.Paint = function(self, w, h)
+		savebutton:SetTextColor(ui_text)
+		surface.SetDrawColor(back_ground.r, back_ground.g, back_ground.b, back_ground.a)
+		surface.DrawRect(0, 0, w, h)
+	end
+
+	savebutton.DoClick = function()
+		if fishingmod.SaveUIColors then
+			fishingmod.SaveUIColors()
+			chat.AddText(Color(0, 255, 0, 255), "[Fishing Mod]", Color(255, 255, 255), ": Colors have been saved!")

```

Comment:
if you're gonna re-use colors, please store them instead of re-creating them, creating tables in lua is intensive. This can be avoided